src/features/shields/structure.ts

```diff
@@ -141,7 +141,10 @@ export const BlockedInfoRowData = styled<BlockedInfoRowDataProps, 'div'>('div')`
   grid-gap: 2px;
   align-items: center;
   pointer-events: ${p => p.disabled && 'none'};
-  opacity: ${p => p.disabled && '0.4'};
+
+  & > *:not(:last-child) {
```

Comment:
Will the text always be the last-child, or can we refer to the actual element selector, e.g.: 

```
& > * { opacity: ... }
& > ${StyledText} { opacity: 1 }
```

nit: Do you need the & if it's at the beginning of the line? At least it's explicit

Reply:
without the `:not` operator stylelint starts complaining on `no-descending-specificity` for a totally unrelated style in `MainToggle`. 

```
 149:3  ✖  Expected selector ".selector73 > *" to come before selector ".selector66 > *:last-child"   no-descending-specificity
```